xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_fr.properties

```diff
@@ -2300,14 +2300,14 @@ admin.ooserver.options.source=Ces options sont configur\u00E9es sur le serveur,
 admin.xwiki.officeimporteradmin.description=Configurer le serveur Office.
 
 admin.presentation=Pr\u00E9sentation
-admin.presentation.description=S\u00E9lectionnez les onglets qui seront affich\u00E9s sur la page et configurez l'ent\u00EAte et le pied de page.
+admin.presentation.description=S\u00E9lectionnez les onglets qui seront affich\u00E9s sur la page et configurez l'en-t\u00EAte et le pied de page.
 admin.panelwizard=Assistant de panneau
 admin.panels.panelwizard.description=Ajouter ou supprimer des panneaux, modifier la disposition des panneaux.
 admin.colorthemes=Th\u00E8mes de Couleur
 admin.colorthemes.description=Param\u00E8tres pour de personnalisation des th\u00E8mes de couleur.
 admin.colorthemes.colibrithemes=Th\u00E8mes Colibri
 admin.colorthemes.flamingothemes=Th\u00E8mes Flamingo
-admin.colorthemes.invalidtheme=La valeur actuelle ({0}) n''est pas correcte. The th\u00E8me de couleur n''existe peut-\u00EAtre pas.
+admin.colorthemes.invalidtheme=La valeur actuelle ({0}) n''est pas correcte. Le th\u00E8me de couleurs n''existe peut-\u00EAtre pas.
```

Comment:
Paul, you've used uppercase "T" for "Thème" in another translation fix you did. Should we use "Thème" instead of "thème" everywhere in your opinion? Thx!